.infrastructure/daemonset.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: todoapp-daemonset
+  namespace: todoapp
+spec:
+  selector:
+    matchLabels:
+      app: busybox
+  template:
+    metadata:
+      labels:
+        app: busybox
+    spec:
+      containers:
+      - name: busybox
+        image: losiev/busyboxplus:curl
+        resources:
+          requests:
+            memory: "32Mi"
+            cpu: "100m"
+          limits:
+            memory: "64Mi"
+            cpu: "250m"
+        args:
+        - /bin/sh
+        - -c
+        - "while true; 
+        do curl http://todoapp-service.todoapp.svc.cluster.local; 
+        sleep 5; done"      
+         
```

Comment:
![image](https://github.com/user-attachments/assets/15c49534-97aa-4d40-a6ed-dc7f3434c103)

Reply:
For EoF to be displayed correctly, the last line should be blank. Remove space
![image](https://github.com/user-attachments/assets/2ce04c20-990d-4373-bb5f-d0cec41a050b)